//When I set config 'port' as '8889'

Not Found

The requested URL was not found on the server. If you entered the URL manually please check your spelling and try again.


//When I set config 'port' as '3306' (assignment default)

  File "/Users/matthewshim/CodingDojo/Python/myEnvironments/py2FlaskEnv/lib/python2.7/site-packages/MySQLdb/connections.py", line 193, in __init__
    super(Connection, self).__init__(*args, **kwargs2)
sqlalchemy.exc.OperationalError: (_mysql_exceptions.OperationalError) (2003, "Can't connect to MySQL server on '127.0.0.1' (61)")